packages/backend/src/extension.ts

```diff
@@ -114,6 +174,15 @@ export async function activate(extensionContext: ExtensionContext): Promise<void
       await openBuildPage(panel, image);
     }),
   );
+
+  macadam = new macadamJSPackage.Macadam(macadamName);
+  await macadam.init();
```

Comment:
Shouldn't we have if statements to only allow on Mac since that's what we support at the moment?

We should also have this in a try / catch since there may be issues initializing that would break monitorMachines as well as the general extension activation.